packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/SingleSeriesConfiguration.tsx

```diff
@@ -86,35 +108,35 @@ const SingleSeriesConfiguration: FC<Props> = ({
                     />
                 )}
                 {isGrouped && (
-                    <Button
-                        icon={series.hidden ? 'eye-open' : 'eye-off'}
+                    <ActionIcon
                         onClick={() => {
                             updateSingleSeries({
                                 ...series,
                                 hidden: !series.hidden,
                             });
                         }}
-                    />
+                    >
+                        <MantineIcon
+                            icon={series.hidden ? IconEye : IconEyeOff}
+                        />
+                    </ActionIcon>
                 )}
                 {isCollapsable && (
-                    <Button
-                        disabled={series.hidden}
-                        icon={isOpen ? 'caret-up' : 'caret-down'}
-                        onClick={toggleIsOpen}
-                    />
+                    <ActionIcon onClick={toggleIsOpen}>
+                        <MantineIcon
+                            icon={isOpen ? IconChevronUp : IconChevronDown}
+                        />
+                    </ActionIcon>
                 )}
             </Group>
-            <SeriesOptionsWrapper
-                isOpen={!isCollapsable || isOpen}
-                $isGrouped={isGrouped}
-                $isSingle={isSingle}
-            >
-                <SeriesExtraInputs>
-                    <SeriesExtraInputWrapper label={!isGrouped && 'Chart type'}>
-                        <SeriesExtraSelect
-                            fill
+            <Collapse in={!isCollapsable || isOpen || false}>
```

Comment:
`|| false` is extra

Reply:
Extra logically, but the prop wants a boolean instead of undefined.